packages/serialize/types/tests.ts

```diff
@@ -37,6 +37,20 @@ serializeStyles(
   {}
 )
 // $ExpectType SerializedStyles
+serializeStyles<{ vars: { background: string; foreground: string } }>([
```

Comment:
we could use a negative~ test case for smth like this:
```ts
serializeStyles({
  // $ExpectError
  display: () => 'unknown'
})
```

Reply:
Good point.

Reply:
I tried but did not work because `csstype` allows the value to be a string so `display: () => 'unknown'` does work.

```js
// node_modules/csstype

export type Display =
    | Globals
    | DataType.DisplayOutside
    | DataType.DisplayInside
    | DataType.DisplayInternal
    | DataType.DisplayLegacy
    | "contents"
    | "list-item"
    | "none"
    | (string & {});
```

Reply:
Oh, ok - so to make such a test case you'd have to find some other property that doesn't allow `string & {}`. If it's hard to find one - let's not bother with this.

Reply:
The `borderCollapse` does not have `string & {}`, added to the test case.